gateway/rpc_storage_handler.go

```diff
@@ -151,12 +151,21 @@ func (r *RPCStorageHandler) Connect() bool {
 
 func (r *RPCStorageHandler) buildNodeInfo() []byte {
 	config := r.Gw.GetConfig()
+	checkDuration := config.LivenessCheck.CheckDuration
+	var intCheckDuration int64
+	if checkDuration == 0 {
+		intCheckDuration = 10
+	} else {
+		// NodeData.TTL expects an int64 value, so we're getting the number of seconds expressed in int64 instead of time.Second
+		intCheckDuration = int64(checkDuration / time.Second)
```

Comment:
what do you think if we move this logic into config? so we call `config.LivenessCheck.CheckDuration()` so we have even the logic of defaulting more centralized

Reply:
done!

I'd still keep the current validation in both functions just in case